clover.rb

```diff
@@ -323,7 +323,13 @@ def assets
     login_label "Email Address"
     two_factor_auth_return_to_requested_location? true
     already_logged_in { redirect login_redirect }
-    after_login { remember_login if request.params["remember-me"] == "on" }
+    after_login do
+      remember_login if request.params["remember-me"] == "on"
+      if omniauth_identity && (url = omniauth_params["redirect_url"])
+        flash["notice"] = "You have successfully connected your account with #{omniauth_provider.capitalize}."
```

Comment:
Is this message still accurate?  With the new code, you connect social identities after login and not during the login process, but maybe that connection process runs though this code?

Reply:
We only set `omniauth_params["redirect_url"]` on the Login Methods page. If it's initiated from the social login buttons on the login form, it won’t have this parameter.

https://github.com/ubicloud/ubicloud/blob/c151a223e464a5445773238119a4aa5856f0eb5a/views/account/login_method.erb#L31